benchmarks/benchmarks.py

```diff
@@ -156,7 +156,7 @@ def setup(self):
             }
         )
 
-        t = ibis.pandas.connect({'df': self.data}).table('df')
+        t = ibis.backends.pandas.connect({'df': self.data}).table('df')
```

Comment:
This is because of something very tricky and counter-intuitive in Ibis, but not correct.

If the file imported `ibis.pandas` (instead of `ibis`), the original `ibis.pandas` in this line would be the `ibis.pandas` module. But Since `import ibis` is used, this is the `pandas` attribute in `ibis/__init__.py` (which is the module, but that's irrelevant). That's quite bad practice IMHO, but what it means is that you need to revert this change.